### ISSUE TITLE
Nuget Package 2.1.3 Broken!

### ISSUE THREAD

**Issue by miketonguk** [docusign/docusign-csharp-client#128]
There appears to be issues with the dependencies in the nuget package. I created a new console app, added the 2.1.3 nuget package then tried to run the sample app code from the main github page. As soon as it hits the line: 

`ApiClient apiClient = new ApiClient("https://demo.docusign.net/restapi");`

you get a runtime error:

```
Exception thrown: 'System.IO.FileLoadException' in PortalTemplate.WebApp.IntegrationTests.dll

Additional information: Could not load file or assembly 'RestSharp, Version=105.0.1.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
```

**Comment by gsnavin** [comment]
@miketonguk Sorry about the inconvenience. I believe this should be fixed with version 2.1.4, can you please try with that version.

**Comment by miketonguk** [comment]
Thanks for the prompt response, however the new version has the same problem, this time the error is:

```
An unhandled exception of type 'System.IO.FileLoadException' occurred in ConsoleApplication1.exe

Additional information: Could not load file or assembly 'RestSharp, Version=100.0.0.0, Culture=neutral, PublicKeyToken=598062e77f915f75' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
```

I've actually tried the last 4 or 5 versions of the Nuget package and they all have a similar issue. In desperation I also tried an unofficial nuget package called 'Docusign' from jkergosi and this one works just fine. We cannot use this in production however.

I also downloaded the code to try and build it myself but the github source doesn't appear to be up to date (last updated 20 days ago, so your update from yesterday is not there). There was also no VS solution file, and the compile.bat in the sdk folder generates a ton of build errors so there seems to be something missing.
